### PR TITLE
Docs backport views

### PR DESCRIPTION
**Pull request by mperezfuster**
Adding documentation for views pg_stat_wal and pg_stat_slru, along with GUC track_wal_io_timing

Review site:
https://docs-staging.vmware.com/en/VMware-Tanzu-Greenplum/mireia-views/greenplum-database/GUID-ref_guide-system_catalogs-pg_stat_wal.html
https://docs-staging.vmware.com/en/VMware-Tanzu-Greenplum/mireia-views/greenplum-database/GUID-ref_guide-system_catalogs-pg_stat_slru.html
https://docs-staging.vmware.com/en/VMware-Tanzu-Greenplum/mireia-views/greenplum-database/GUID-ref_guide-config_params-guc-list.html#track_wal_io_timing

Please let me know if the content is correct, especially the columns of view pg_stat_wal as I had to remove some references to undocumented content (see https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STAT-WAL-VIEW for reference).

### CODE REVIEW

File: gpdb-doc/markdown/ref_guide/system_catalogs/pg_stat_wal.html.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# pg_stat_wal 
+
+The view `pg_stat_wal` shows data about the WAL activity of the cluster. It contains always a single row.
+
+|column|type|references|description|
+|------|----|----------|-----------|
+|`wal_records`|bigint| |Total number of WAL records generated.|
+|`wal_fpw`|bigint| |Total number of WAL full page images generated.|
```
Comment (huansong) [inline gpdb-doc/markdown/ref_guide/system_catalogs/pg_stat_wal.html.md:8]: Great catch on the name difference!

File: gpdb-doc/markdown/ref_guide/system_catalogs/pg_stat_slru.html.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# pg_stat_slru
+
+Greenplum Database accesses certain on-disk information via SLRU (simple least-recently-used) caches. The `pg_stat_slru` view contains one row for each tracked SLRU cache, showing statistics about access to cached pages.
+
```
Comment (huansong) [inline gpdb-doc/markdown/ref_guide/system_catalogs/pg_stat_slru.html.md:4]: Not sure if we should mention something like this: https://docs.vmware.com/en/VMware-Tanzu-Greenplum/6/greenplum-database/GUID-ref_guide-system_catalogs-pg_stat_indexes.html#:~:text=display%20access%20statistics%20for%20indexes%20only%20from%20the%20master%20instance . Same situation for `pg_stat_slru`/`pg_stat_wal`: only coordinator view unless using `gp_dist_random`. But seems that it's not always mentioned: https://docs.vmware.com/en/VMware-Tanzu-Greenplum/6/greenplum-database/GUID-ref_guide-system_catalogs-pg_stat_tables.html . I guess we can skip for now.

File: gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md [hunk 3]
```diff
@@ -2,36 +2,38 @@
 
 Greenplum Database provides the following system views not available in PostgreSQL.
```
Comment (huansong) [inline gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md:3]: A lot of the views in the list below _are_ available in PG (including the two we are adding).

There's a link to the PG doc of system views under `For more information about the standard system views supported in PostgreSQL and Greenplum Database, see the following sections of the PostgreSQL documentation`. That link is PG12 for GPDB7, and PG9.4 for GPDB6.

So it seems that we should really say something like `Greenplum 7 Database provides the following system views not available in PostgreSQL 12` here.

But still there's `pg_stat_activity`, `pg_stat_all_indexes` and `pg_stat_all_tables` that are in PG12 as well. Should we remove them?

**Review by huansong — APPROVED** [review]
LGTM besides one change needed about the description of the list of views in our doc.

File: gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md (hunk 3 above)
Comment (mperezfuster) [reply gpdb-doc/markdown/ref_guide/system_catalogs/catalog_ref-views.html.md:3]: @dyozie this is actually an interesting point, we document many postgresql views on this page so should we change the description?

File: gpdb-doc/markdown/ref_guide/system_catalogs/pg_stat_slru.html.md (hunk 2 above)
Comment (mperezfuster) [reply gpdb-doc/markdown/ref_guide/system_catalogs/pg_stat_slru.html.md:4]: I can update it for all relevant views, can you let me know which ones need to be updated? Thanks.

**Review by dyozie — APPROVED** [review]

**Pull request merged**